recipe/meta.yaml

```diff
@@ -24,96 +24,186 @@ requirements:
     - wheel
   run:
     - python
-    - datasets !=2.5.0
+    # We need either this pin or pyarrow<21.0.0
+    - datasets >=2.15.0
     - filelock
-    - huggingface_hub >=0.30.0,<1.0
+    - huggingface_hub >=0.34.0,<1.0
     - numpy >=1.17
     - packaging >=20.0
     - pyyaml >=5.1
     - regex !=2019.12.17
     - requests
     - safetensors >=0.4.3
-    - tokenizers >=0.21,<0.22
+    - tokenizers >=0.22.0,<=0.23.0
     - tqdm >=4.27
-    - protobuf
   run_constrained:
-    - blas * openblas     # [osx and x86_64]
-    # Extra dependencies by group
-    # Torch related
-    - pytorch >=2.0   # extra:torch
-    - accelerate >=0.26.0   # extra:accelerate
-    # TensorFlow related
-    - tensorflow >2.9,<2.16   # extra:tf
-    - tensorflow-text <2.16   # extra:tf
-    - keras-nlp >=0.3.1,<0.14.0     # extra:tf
-    - tensorflow-cpu >2.9,<2.16   # extra:serving
-    - keras >2.9,<2.16 # extra:serving
-    - tensorflow-probability <0.24   # extra:serving
-    # Flax related
-    - flax >=0.4.1,<=0.7.0   # extra:flax
-    - jax >=0.4.1,<=0.4.13 # extra:flax
-    - jaxlib >=0.4.1,<=0.4.13 # extra:flax
-    - optax >=0.0.8,<=0.1.4 # extra:flax
-    - scipy <1.13.0   # extra:flax
-    # Agents related
-    - datasets !=2.5.0 # extra:agents
-    - sentencepiece >=0.1.91,!=0.1.92   # extra:agents
-    # Vision related
-    - Pillow >=10.0.1,<=15.0   # extra:vision
-    # Japanese related
-    - fugashi >=1.0 # extra:ja
-    - ipadic >=1.0.0,<2.0 # extra:ja
-    - unidic_lite >=1.0.7 # extra:ja
-    - unidic >=1.0.2 # extra:ja
-    - sudachipy >=0.6.6 # extra:ja
-    - sudachidict_core >=20220729 # extra:ja
-    - rhoknp >=1.1.0,<1.3.1   # extra:ja
-    # ONNX related
-    - onnxruntime >=1.4.0 # extra:onnxruntime
-    - onnxruntime-tools >=1.4.2   # extra:onnxruntime
-    # Miscellaneous
-    - timm <=1.0.11     # extra:timm
-    - codecarbon >=2.8.1     # extra:codecarbon
-    - av ==9.2.0   # video
-    - ray-tune >=2.7.0
-    - kernels >=0.3.2,<0.4   # extra:hub-kernels
-    - ray >=2.7.0   # extra:ray
-    - deepspeed >=0.9.3   # extra:deepspeed
-    - cookiecutter ==1.7.3   # extra:modelcreation
-    - natten >=0.14.6,<0.15.0   # extra:natten
-    - sagemaker >=2.31.0   # extra:sagemaker
-    - huggingface-hub >=0.30.0,<1.0 # extra:tiktoken
-    - pyctcdecode >=0.4.0 # extra:tf-speech
+    # Duplicates are commented to avoid build errors!!!
+    # ja(Japanese) related
+    - fugashi >=1.0 
+    - ipadic >=1.0.0,<2.0 
+    - unidic_lite >=1.0.7 
+    - unidic >=1.0.7
+    - sudachipy >=0.6.6 
+    - sudachidict_core >=20220729 
+    - rhoknp >=1.1.0,<1.3.1   
+    # sklearn related
+    - scikit-learn
```

Comment:
Optional dependencies without version constraints aren't necessary in run_constrained.